### PR TITLE
Allow use of go-httpbin as a command

### PR DESCRIPTION
**Pull request by andrewgaul**

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
@andrewgaul thanks for your patch. I just reviewed this. This appears to be a breaking change. 

Instead, can this be done by creating a cmd/httpbin/main.go? That should be as simple as 1 file change and it is backwards-compatible.

**Comment by andrewgaul** [comment]
Done.  I am a little confused about how go manages commands in subpackages since you need to change into the directory to build it.

File: README.md [hunk 1]
```diff
@@ -87,6 +87,16 @@ func TestDownload(t *testing.T) {
 }
 ```
 
+go-httpbin works from the command line as well:
+
+```
+$ pwd
+$GOPATH/src/github.com/ahmetalpbalkan/go-httpbin
```
Comment (ahmetalpbalkan) [inline README.md]: can you just change this to:

> To download go-httpbin executable directly, run:
>
>     github.com/ahmetalpbalkan/go-httpbin/cmd/httpbin
>
> and execute `$GOPATH/bin/httpbin -host :8080`.

File: httpbin/main.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package main
```
Comment (ahmetalpbalkan) [inline httpbin/main.go]: can you please move this file to cmd/httpbin? It's the idiomatic source tree structure in Go.

Comment (andrewgaul) [inline httpbin/main.go]: Done.

File: README.md (hunk 1 above)
Comment (andrewgaul) [inline README.md]: Done.

**Pull request merged**